Realm/RLMRealm.h

```diff
@@ -19,7 +19,14 @@
 #import <Foundation/Foundation.h>
 #import "RLMConstants.h"
 
-@class RLMRealmConfiguration, RLMObject, RLMSchema, RLMMigration, RLMNotificationToken, RLMThreadSafeReference;
+@class RLMRealmConfiguration, RLMRealm, RLMObject, RLMSchema, RLMMigration, RLMNotificationToken, RLMThreadSafeReference;
+
+/**
+ A callback block for opening Realms asynchronously.
+
+ Returns the Realm if the open was successful, or an error otherwise.
+ */
+typedef void(^RLMAsynchronouslyOpenRealmCallback)(RLMRealm * _Nullable realm, NSError * _Nullable error);
```

Comment:
Well, it'd be `RLMAsyncOpenRealmCallback` since the new method name is "async open" not "open async". But sure.